Seti.tmTheme

```diff
@@ -444,7 +444,7 @@
                 <key>settings</key>
                 <dict>
                     <key>foreground</key>
-                    <string>#009FFF</string>

```

Comment:
Or better keep it. This line belongs to the scope: **keyword.control.import.python** I feel the import section needs a distinct color as well. Also good to note stray import statements throughout the code, like those in pdb breakpoint (import pdb, pdb.set_trace())